src/app/components/lunch/lunch.component.ts

```diff
@@ -126,5 +126,7 @@ export class LunchComponent implements OnInit {
 
 		return dates;
 	}
-
+	lunchClassMaker(classInput) {
+		return classInput.toLowerCase().replace(/ /, "-");
```

Comment:
I'm pretty sure this should be a global regex (`/ /g`), otherwise, it will only replace the first occurrence of a space with a hyphen.